tools/spectrast/spectrast_import.xml

```diff
@@ -0,0 +1,44 @@
+<tool id="spectrast_import" name="SpectraST Import" version="5.0.0">
+    <description>Spectral Library</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        ## Symlink dataset to name with recognized ext
+        #import re
+        #set $ext = '.' + str($input.extension)
+        #set $input_name = $re.sub('[.](ms2|msp|hlf)$','',$input.display_name.split('/')[-1]) + $ext
```

Comment:
you could use $input.element_identifier here ...

Reply:
@bgruening   I'm not familiar with the element_identifier property.  Are there some good examples I should view?

Reply:
Here for example: https://github.com/galaxyproject/tools-iuc/blob/7036343b9ac0a0ffc2ce4f6db465b9298ef05e73/tools/picard/picard_macros.xml#L21